src/frontend/src/tests/app/AppAuthenticated.test.tsx

```diff
@@ -15,6 +19,14 @@ jest.mock('../../pages/ProjectsPage/Projects', () => {
   };
 });
 
+jest.mock('../../hooks/misc.hooks');
```

Comment:
instead of doing this whole thing which takes up way too much space, look at the `mock-hooks` file adn how ive used that in some places. it's much easier.